docs/how-to-guides/themes/theme-json.md

```diff
@@ -152,12 +146,68 @@ This specification is the same for the three different origins that use this for
 
 ### Version
 
-This field describes the format of the `theme.json` file and it's used to detect different versions and migrate them to the latest format.
+This field describes the format of the `theme.json` file. The current and only version is 1.
+
+WordPress 5.8 will ignore the contents of any `theme.json` whose version is not equals to the current. Should the Gutenberg plugin need it, it'll update the version and will add the corresponding migration mechanisms from older versions.
 
 ### Settings
 
+<div class="callout callout-alert">
+The Gutenberg plugin extends the settings available from WordPress 5.8, so they can be used with other WordPress versions and they go through a maturation process before being ported to core.
+
+The tabs below show WordPress 5.8 supported settings and the ones supported by the Gutenberg plugin.
+</div>
+
 The settings section has the following structure:
 
+{% codetabs %}
+{% WordPress %}
+
+```json
+{
+	"version": 1,
+	"settings": {
+		"color": {
+			"custom": true,
+			"customGradient": true,
+			"duotone": [],
+			"gradients": [],
+			"link": false,
+			"palette": []
```

Comment:
It may be helpful to define a preset like a color so people know how to do it.

Reply:
There's a section fully focused on presets.